modules/database/main.tf

```diff
@@ -2,8 +2,10 @@
 # SPDX-License-Identifier: MPL-2.0
 
 resource "random_string" "postgresql_password" {
-  length  = 128
-  special = false
+  length           = 128
+  special          = true
+  min_special      = 4
+  override_special = "+(!]&;$)"
```

Comment:
❓ Could you please share the reason behind these specific characters for `override_special`? This looks like a learning opportunity to me.

Reply:
The [override_special setting](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/string#override_special) indicates exactly which special characters to include in the random string that is generated (for example, there is no "@" symbol and so that special char will not be included in the output string).